docs/topics/contributing-changing-existing-content.adoc

```diff
@@ -0,0 +1,67 @@
+:docs-topic: <3>
+
+= {contrib-guide-name} <4>
+
+
+== {launcher} Development
+
+This chapter contains information about developing your own runtimes or boosters, contributing changes to existing ones, and improving {launcher}.
+
+include::topics/assembly-updating-booster-catalog-for-local-minishift-testing.adoc[leveloffset=+2] <5>
+
+You can preview the latest state of {launcher} by navigating to the link:{launcher-stage}[stage build] of {launcher}.
+
+== Filing a Code Issue
+...
+----
+<1> This line includes a common file that sets common attributes used across all guides.
+<2> This line is a comment that will not be rendered in the final output.
+<3> This line defines a new attribute called `docs-topic`
+<4> This line sets the document title, which happens to use the existing attribute `contrib-guide-name`. This attribute is defined in `docs/topics/templates/document-attributes.adoc`.
+<5> This line adds the content from `topics/assembly-updating-booster-catalog-for-local-minishift-testing.adoc` and adds a link:http://asciidoctor.org/docs/user-manual/#include-partitioning[level offset] of 2.
+
+There are a few important concepts that are heavily used that you need to learn:
+
+document-attributes file::
+  This file (`docs/topics/templates/document-attributes.adoc`) sets common variables that are used across all the guides. Every guide includes this file.
+partitioning content::
+  Sections of content in each guide are broken out into smaller files. This allows for content to be easily refactored within a guide as well as reused across different guides. The `include` directive is used to include these files within a piece of content.
+modular documentation::
+  An individual piece of content, usually called a module, is formatted in xref:writing-the-modular-way[a modular manner], which allows for better re-use across guides.
+common topics directory::
+  All of the smaller content files and modules are located in the `docs/topics/` directory. This is a common directory that contains _all_ shared files across all guides. To help the build process, the `docs/topics/` directory is symlinked in each guide's directory.
+
+== Basic Workflow for Modifying Existing Guides
+
+[discrete]
+=== Prerequisites
+
+* An IDEs for editing asciidoc, such as link:https://atom.io[the Atom editor].
+
+[discrete]
+=== Procedure
+
+. Find the guide you want to modify and open it's `master.adoc` file.
+. Find the section that you want to modify and open up the appropriate content file. 
++
+NOTE: Many IDEs have a convenient way to quickly open a file by name. For example, link:https://atom.io[the Atom editor] has a nice _Find File_ command that allows you to quickly search by filename and path.
+
+. Make your edits.
+. Verify your changes by doing a xref:_building_locally[local build] and viewing the HTML.
+. Commit and submit your changes using the xref:git-workflow-making-changes[Git workflow]. 
```

Comment:
I would suggest removing _Commit_ because it's a Git term, whereas _submit_ isn't, so it could cause confusion.

_using Git workflow_ seems clunky to me. I suggest replacing with _according to instructions in xref:git-workflow-making-changes[]._

Reply:
fixed